### PR TITLE
Update minimum node version to 16+

### LINKED ISSUE #16764: directus isn't compatible with node v13.14.0
### Describe the Bug

Directus is required at Node 12.20 or later。but it isn't compatible with node v13.14.0。

### Errors Shown

_No response_

### What version of Directus are you using?

9.21.2

### What version of Node.js are you using?

13.14.0

### What database are you using?

sqlite3  5.1.2

### What browser are you using?

chrome

### How are you deploying Directus?

locally

### PR DESCRIPTION
**Pull request by Nitwel**
closes: #16764

### CODE REVIEW

**Comment by rijkvanzanten** [comment]
Elsewhere in the docs, we note current LTS as the required version. That'd be 18 at the time of writing :)

**Comment by Nitwel** [comment]
Done

File: readme.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ important to keep your projects up-to-date.
 
 Directus only requires Node.js and supports most operating systems and SQL database vendors.
 
-- Node.js 12.20+
+- Node.js 18+
```
Comment (rijkvanzanten) [inline readme.md:61]:
```suggestion
- Node.js [active LTS](https://nodejs.dev/en/about/releases/)
```

**Comment by piotr-cz** [comment]
I understand that the requirement should be bumped to node v14 (even that formally it would be a breaking change) but jumping straight to v18 when not required is IMHO too much.

For example Node 18 support AWS Lambda has been added just [few weeks ago](https://github.com/aws/aws-lambda-base-images/issues/47#issuecomment-1320395177).

**Review by br41nslug — APPROVED** [review]

**Pull request merged**